templates/logrotate.d/jitsi-videobridge.j2

```diff
@@ -0,0 +1,10 @@
+/var/log/jitsi/jvb.log {
+  daily
```

Comment:
The videobridge is rotated hourly in the default and it seems that is needed. The service logs a lot.

Reply:
what about `{ jitsi_meet_logrotate_retained_days *24 }}` then? and related, it seems that `jitsi_meet_videobridge_loglevel` has no effect, doesn't it? i see entries about health checks at info level every ten seconds.

Reply:
`/etc/jitsi/videobridge/logging.properties` defines a console logger.

Reply:
> i see entries about health checks at info level every ten seconds.

i'm being told that this behaviour was a bug and has been fixed in a recent release. but i still see this with the most recent `2.1-197-g38256192-1`. could something from the role be the cause?

Reply:
> i'm being told that this behaviour was a bug and has been fixed in a recent release. but i still see this with the most recent 2.1-197-g38256192-1. could something from the role be the cause?

does #29 help?

Reply:
that affects jicofo, not the videobridge, doesn't it?

Reply:
okay. there are now other diffs to latest upstream release.

Reply:
I also incorporated this patch, but without jitsi-videobridge. Their logs are *just to big*